### ISSUE TITLE
Error when running examples.ipynb

### ISSUE THREAD

**Issue by xychen9459** [szagoruyko/pytorchviz#4]
When I run examples.ipynb, notebook shows me the following error message:

```
trace, _ = torch.jit.trace(model, args=(x,))
make_dot_from_trace(trace)

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-95b4d4bfd5bf> in <module>()
      1 trace, _ = torch.jit.trace(model, args=(x,))
----> 2 make_dot_from_trace(trace)

/opt/anaconda-py/anaconda3/lib/python3.6/site-packages/torchviz/dot.py in make_dot_from_trace(trace)
    108     >>> dot = make_dot_from_trace(trace)
    109     """
--> 110     torch.onnx._optimize_trace(trace, False)
    111     graph = trace.graph()
    112     list_of_nodes = parse(graph)

TypeError: _optimize_trace() takes 1 positional argument but 2 were given
```

How to solve the problem?

**Comment by eugenelet** [comment]
Same problem here, any solution to this?

**Comment by Jongchan** [comment]
@eugenelet Does it work without the ```False``` argument?

**Comment by derelearnro** [comment]
Also me the same problem, 

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-fe552e8dc282> in <module>()
----> 1 trace, _ = torch.jit.trace(model, args=(x,))
      2 make_dot_from_trace(trace)

TypeError: 'function' object is not iterable

**Comment by szagoruyko** [comment]
fixed by #13
